en/flight_controller/pixhawk4_mini.md

```diff
@@ -145,12 +145,12 @@ UART8 | /dev/ttyS6 | Not connected (no PX4IO)
 
 
 ## Supported Platforms
-Motors and servos are connected to the **MAIN OUT** ports in the order specified for your vehicle in the [Airframe Reference](../airframes/airframe_reference.md). 
+Motors and servos are connected to the **MAIN OUT** ports in the order specified for your vehicle in the [Airframe Reference](../airframes/airframe_reference.md).
```

Comment:
@vincentpoont2 You probably know this, but we may have to modify this terminology in the next version of PX4. Beat Kung did this new thing that allows you to map outputs whereever you like using QGC. 

Just a heads up for now. We might not need to change this.